metrics/collectors/average.lua

```diff
@@ -0,0 +1,59 @@
+local fiber = require('fiber')
+
+local Shared = require('metrics.collectors.shared')
+
+--- Collector to produce count and average value metrics.
+-- Average value is is calculated between two subsequent `:collect` calls.
+local Average = Shared:new_class('average')
+
+function Average:new(name, help)
+    local obj = Shared.new(self, name, help)
+    obj.count_name = name .. '_count'
+    obj.avg_name = name .. '_avg'
```

Comment:
should we also export `name .. '_sum'` value?

Reply:
I can add. What is the use case?